### ISSUE TITLE
Pyright suddenly reporting a lot more errors

### ISSUE THREAD

**Issue by SHxKM** [microsoft/pyright#296]
**Describe the bug**
I'm not sure this is a bug, however, pyright suddenly started reporting *a lot* more errors on my Django project. Previously, pyright would only report errors when I have an annotated a certain method or parameter and was using going against the annotation. However, pyright seems more eager right now, and is reporting errors for `models` file for example:

```python
class UserProfile(BaseModel):

    user = models.OneToOneField(
        User, related_name="user_profile", on_delete=models.CASCADE
    )
    pending_email = models.EmailField(null=True)
```

<img width="520" alt="Screen Shot 2019-10-21 at 22 35 12" src="https://user-images.githubusercontent.com/6573474/67237320-b35a1900-f453-11e9-9197-d360df65467f.png">

**Expected behavior**
Previously pyright would only point out errors where explicitly annotated functions where given. For example:

```python
def foo(self) -> str:
    return self.name

g = 1 + c.foo() - 5 # example

```

**Comment by SHxKM** [comment]
Note: this doesn't occur in version `1.0.75`.

**Comment by erictraut** [comment]
Thanks for the bug report. This same issue was reported by a couple of people. It's due to a regression that I introduced in 1.0.77. I'll have it fixed by the end of the day. Sorry for any inconvenience.

Incidentally, the problem doesn't occur if you have type stubs for the libraries that you're importing.

**Comment by erictraut** [comment]
I just published 1.0.79, which addresses this problem. Thanks again for reporting it.
